VariablesHomework/VariablesHomework.cs

```diff
@@ -148,31 +148,32 @@ public void Conversion()
         {
             int i = 13;
             double d = i;
-            Assert.IsTrue(d == /*Answer here*/); // Make this pass
+            Assert.IsTrue(d == 13); // Make this pass
 
-            // Why does that compile andwork but this doesnt?
+            // Why does that compile and work but this doesnt?
             // double d = 15.5;
             // int c = d;
+            // Answer: double can't convert to integer because is less accurate 
         }
 
         [TestMethod]
         public void Casting()
         {
             // what am i doing here that make the above work?
             double d = 15.5;
-            int c = (int)d;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass
+            double c = d;
```

Comment:
Don't change this line. It should be `int c = (int)d;`

The question is why _does_ this work? (We haven't talked about the answer yet)